config/default.yml

```diff
@@ -293,3 +293,15 @@ Sorbet/ValidSigil:
   - bin/**/*
   - db/**/*.rb
   - script/**/*
+
+Sorbet/ValidGemVersionAnnotations:
+  Description: >-
+                  Ensures all gem version annotations in RBI files are correctly formatted per
+                  Ruby's gem version specification guidelines.
+
+                  See the rubygems.org documentation for more information on how to format gem
+                  versions: https://guides.rubygems.org/patterns/#pessimistic-version-constraint
+  Enabled: true
```

Comment:
I'm not sure it safe to enable by default since shims or generated gem RBIs may have surprising comments.

Don't we intend Tapioca to strip the version tagging when pulling annotations? I think we also talked about the possibility to merge annotations directly into the generated gem RBIs right?